eslint-common.json

```diff
@@ -9,6 +9,7 @@
   "dot-location": ["property"],
   "eol-last": ["always"],
   "eqeqeq": ["always", {"null": "never"}],
+  "for-direction": [],
```

Comment:
That's not actually the case:

```make
eslint-%.json: eslint-%-only.json eslint-common.json scripts/eslint-combine
	scripts/eslint-combine eslint-common.json '$<' >'$@'
```

Running `make` updates both derived files based on changes to __eslint-common.json__. The script which combines the common rules and the ES3/ES6 rules sets the level to `"error"` in all cases, to save us from having to specify it for every rule. :)